util/tracing/env/traceenv.go

```diff
@@ -14,10 +14,10 @@ const (
 )
 
 func init() {
-	appcontext.Register(initContext)
+	appcontext.Register(InitContext)
 }
```

Comment:
Wondering if we need to keep the `init`, as that would be dead-code if `appcontext` is not used (but it will remain a dependency due to it being imported) 🤔 

(have not looked how / where it's imported though).

Reply:
Yeah, I thought about making other changes to `appcontext`/this package for that but had concerns about that possibly silently breaking things for other downstream users. If we do that, we should do that together with other changes to make sure we don't silently break, or clearly call it out.

Reply:
(side note, but I really hate `init`s for this reason, it's much harder to reason about and figure out the blast radius of changes like this)